README.md

```diff
@@ -59,87 +51,90 @@ After this, you have some options:
     > salloc -N 1 -t 60 --account=mpafess --partition=debug
     # ... Wait for a compute node
     > cd HolisticElectricityModel.jl
-    > module load gurobi/9.1.2
+    > module load gurobi
     > julia --project=runner script/driver.jl
     ```
 
+#### NREL Laptop Instructions
 
-### Run Style
+- Go to https://julialang.org/downloads/ and isntall Julia based on the OS.
+- [Install the Xpress solver](https://github.nrel.gov/MSOC/fico-xpress)
+- Use the driver_xpress.jl file for your model runs
```

Comment:
`driver_xpress.jl` has been removed

Reply:
Removed the scripts as well as references from the readme.